src/Shared/Configuration/OpenTelemetryConfigurationExtensions.cs

```diff
@@ -107,6 +107,24 @@ internal static class OpenTelemetryConfigurationExtensions
         return true;
     }
 
+    public static bool TryGetStringValue(
```

Comment:
This isn't logging anything why not just call https://github.com/open-telemetry/opentelemetry-dotnet/blob/e5ca93b8f9008a55dc23ddfca45f7013fa4f7287/src/Shared/Configuration/OpenTelemetryConfigurationExtensions.cs#L23?

Reply:
Fixed.